shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -794,7 +779,6 @@ @implementation FlutterSemanticsObject {
 
 // Method declared as unavailable in the interface
```

Comment:
likely not related to this pr - can we just remove this `init` since it's already marked as unavailable?

Reply:
Good idea.